docs/user/ppl/index.rst

```diff
@@ -110,6 +110,8 @@ The query start with search command and then flowing a set of command delimited
 
   - `reverse command <cmd/reverse.rst>`_
 
+  - `table command <cmd/table.rst>`_
```

Comment:
Instead of a separate doc we could also just have this link into `fields`, not too important though.